substrate/frame/balances/Cargo.toml

```
@@ -24,11 +24,11 @@ frame-support = { workspace = true }
 frame-system = { workspace = true }
 sp-runtime = { workspace = true }
 docify = { workspace = true }
+sp-core = { workspace = true, default-features = true }
```

```suggestion
sp-core = { workspace = true }
```

Sadly we cant enable default features here. I assume that you needed this for the key derivation to work?

---

Verifying this with submit